packages/web-pkg/src/composables/actions/files/useFileActionsOpenShortcut.ts

```diff
@@ -34,12 +36,20 @@ export const useFileActionsOpenShortcut = ({ store }: { store?: Store<any> } = {
   const handler = async ({ resources, space }: FileActionOptions) => {
     try {
       const fileContents = (await clientService.webdav.getFileContents(space, resources[0])).body
-      const url = extractUrl(fileContents)
+      let url = extractUrl(fileContents)
+
+      // Add protocol if missing
+      url = url.match(/^http[s]?:\/\//) ? url : `https://${url}`
 
       // Omit possible xss code
-      const sanitizedUrl = DOMPurify.sanitize(url, { USE_PROFILES: { html: true } })
+      url = DOMPurify.sanitize(url, { USE_PROFILES: { html: true } })
+
+      if (url.startsWith(configurationManger.serverUrl)) {
```

Comment:
will the serverUrl always be defined, i ask because if the serverUrl is an empty string it would enable any url to pass :)

Reply:
always there

Reply:
But it's not sever url 😑